helpers/filterSeats.js

```diff
@@ -23,6 +26,12 @@ module.exports = function(dust) {
     var roomRates = params.roomRates;
     var bandColours = params.bandColours;
     var sortByInput = params.sortBy;
+    var seatLegend = params.seatLegend.seatLegend.en;
```

Comment:
Agreed, this was the intention but had forgotten to make this change before pushing.